thumbor/handlers/__init__.py

```diff
@@ -620,6 +620,8 @@ def _fetch(self, url):
             is_mixed_no_file_storage = is_mixed_storage and isinstance(storage.file_storage, NoStorage)
 
             if not (is_no_storage or is_mixed_no_file_storage):
+                self.context.config = copy.deepcopy(self.context.config)
+                self.context.config.PRESERVE_EXIF_INFO = True
```

Comment:
@masom: nice idea, but the big problem with that is that that requires changing your API for plugins as they stand. Since these aren't even all in one repository, that, as @Savar says is a major piece of work, and one that doesn't make sense for something which, as you rightly point out, is a bug, introduced 4 years ago. Since "done is better than perfect", it makes sense to merge this change now, and think more carefully about long-term APIs for the transform plugins.